properties.schema

```diff
@@ -198,6 +198,16 @@
       "required": false,
       "title": "Feedback",
       "properties": {
+        "title": {
+          "type": "string",
+          "required": false,
+          "default": "",
+          "title": "Feedback Title",
+          "inputType": "Text",
+          "validators": [],
+          "help": "This text will display as the feedback title",
```

Comment:
Could you change this to "Leave blank to have the component's title shown instead."

Reply:
done!